src/lib/stepper/step-header.scss

```diff
@@ -38,10 +38,13 @@ $mat-step-header-icon-size: 16px !default;
   transform: translate(-50%, -50%);
 }
 
-.mat-step-icon .mat-icon {
-  font-size: $mat-step-header-icon-size;
-  height: $mat-step-header-icon-size;
-  width: $mat-step-header-icon-size;
+.mat-step-icon,
+.mat-step-icon-not-touched {
```

Comment:
I believe this class has since been removed

Reply:
We should remove it then, because the class is never actually applied to nay elements

Reply:
Done.

Reply:
Also this ended up being reduced to just removing that extra selector. I can close it if necessary.